scheduler/pkg/tracing/provider_test.go

```diff
@@ -106,11 +106,11 @@ func TestLoadConfig(t *testing.T) {
 		},
 		{
 			name:   "enabled",
-			config: `{"enable":true, "otelExporterEndpoint":"0.0.0.0:1234","ratio":0.5}`,
+			config: `{"enable":true, "otelExporterEndpoint":"0.0.0.0:1234","ratio":"0.5"}`,
 			expectedConfig: &TracingConfig{
 				Enable:               true,
 				OtelExporterEndpoint: "0.0.0.0:1234",
-				Ratio:                0.5,
+				Ratio:                "0.5",
```

Comment:
add test for invalid ratio?

Reply:
added test to loading config and added go test for this check